src/commands/misc/lorem.js

```diff
@@ -0,0 +1,34 @@
+const { SwitchbladeEmbed, Command } = require('../../')
+const { LoremIpsum } = require('lorem-ipsum')
+
+module.exports = class Lorem extends Command {
+  constructor (client) {
+    super({
+      name: 'lorem',
+      parameters: [{
+        type: 'number',
+        full: true,
+        missingError: 'commands:lorem.noNumber',
+        required: false,
+        min: 1,
+        max: 10
+      }]
+    },
+    client)
+
+    this.lorem = new LoremIpsum()
+  }
+
+  async run ({ channel, t }, paragraphs) {
+    let loremString = this.lorem.generateParagraphs(paragraphs || 2)
```

Comment:
```suggestion
  async run ({ channel, t }, paragraphs = 2) {
    let loremString = this.lorem.generateParagraphs(paragraphs)
```

Reply:
This was not resolved

Reply:
Why so? I changed it :D

Reply:
You did not

Reply:
Oh I guess it got undo while I was stuck on rebasing :(

Reply:
I have done the changes, you can now review ;)